src/entities/Tile.ts

```diff
@@ -23,8 +23,8 @@ export class Tile {
 
     public apply(state: TileState, dto: Nullable<TileDTO>): void {
         this.state = state;
-        this.rotation = dto?.rotation || null;
```

Comment:
This is the same problem with the coordinates beeing 0 would result in the rotation and xoordinates to become null.

Reply:
true! i love your use of the nullish coalescing operator (??) which only overwrites it when its null or undefined aka not 0. Note: you can leave the null instead of 0 as default. (desired is that the value is null if the dto is null and the value of the dto (int) otherwise)

Reply:
Background: if the dto is something the value will be a int and it should be used for the tile value. if the dto is null then the ?. will have the effect that the value is undefined and because we dont use undefined we want to exchange this with null (?? null)

Reply:
Oh yeah, my fault. Was just happy it was working 😂